apps/meteor/tests/end-to-end/api/livechat/18-rooms-ee.ts

```diff
@@ -165,6 +166,37 @@ import { IS_EE } from '../../../e2e/config/constants';
 			const updatedRoom = await getLivechatRoomInfo(room._id);
 			expect(updatedRoom.onHold).to.be.true;
 		});
+		it('Should put room on hold, even in the visitor sent the last message', async () => {
+			const { room } = await startANewLivechatRoomAndTakeIt();
+			await updateSetting('Livechat_allow_manual_on_hold_upon_agent_engagement_only', false);
+			const visitor = await createVisitor();
```

Comment:
This will create a different visitor than the one on the room. Actually, this should fail 🤔 

Visitor is returned from the "startNewLivechatRoomAndTakrIt" call